crew

```diff
@@ -817,7 +817,12 @@ def install
     meta = download
     target_dir = unpack meta
     if meta[:source] == true
-      abort "You don't have a working C compiler. Run `crew install buildessential` to get one and try again.".lightred unless system('gcc --version')
+      abort "You don't have a working C compiler. Execute `crew install buildessential` and try again.".lightred unless system('gcc --version')
+      abort "You don't have a working Clang compiler. Execute `crew install buildessential` and try again.".lightred unless system('clang --version')
+      abort "You don't have a working CMake compiler. Execute `crew install buildessential` and try again.".lightred unless system('cmake --version')
```

Comment:
CMake isn't a compiler it's a build system

Reply:
According to Wikipedia, "CMake is not a build system but rather it generates another system's build files."  The actual cmake command does compile source code.

Reply:
The lines above are there to remove the need to add `depends_on 'something' => :build` to all the packages.  In other words, you won't be able to compile anything until buildessential is installed.

Reply:
Alright. According to the cmake website, it's a metabuild system